posts/2024-08-26-webassembly-targets-and-new-on-by-default-features.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: post
+title: "WebAssembly targets and new on-by-default features"
+author: Alex Crichton
```

Comment:
We should mention that you're writing this blog post on the behalf of the compiler team 
```suggestion
author: Alex Crichton
team: The Compiler Team <https://www.rust-lang.org/governance/teams/compiler>
```